### PR TITLE
Integrate the SQS -> MQ lambda into the deployment pipeline

### PR DESCRIPTION
**Pull request by TheGrantsta**
Amends the SQS queue policy to allow lambda

Creates SQS trigger to lambda

Create lambda function that decrypts message and sends the body to HEARING_OUTCOME_INPUT_QUEUE

Logs result to CloudWatch

### CODE REVIEW

File: terraform/stack_infra_next/main.tf [hunk 1]
```diff
@@ -156,8 +156,14 @@ module "sqs-ingress" {
   source = "../../terraform_modules/sqs-ingress"
 
   name_prefix         = module.label.id
+  region_name         = data.aws_region.current_region.name
   tags                = module.label.tags
```
Comment (brettminnie) [inline terraform/stack_infra_next/main.tf:160]: For future reference as a convention, tags should be at the bottom of a resource block on their own line

File: terraform_modules/sqs-ingress/main.tf [hunk 2]
```diff
@@ -93,3 +109,136 @@ resource "aws_sqs_queue_policy" "sendMessage_from_another_account" {
   queue_url = aws_sqs_queue.input_queue[0].id
   policy    = data.aws_iam_policy_document.sendMessage.json
 }
+
+resource "aws_iam_role" "sqs_lambda_role" {
+  name = local.lambda_role_name
+
+  assume_role_policy = <<EOF
```
Comment (brettminnie) [inline terraform_modules/sqs-ingress/main.tf:116]: If you write this as <<-EOF you can indent like a real person ;)

File: terraform_modules/sqs-ingress/main.tf [hunk 3]
```diff
@@ -73,6 +78,7 @@ data "aws_arn" "this" {
 }
 
 data "aws_iam_policy_document" "sendMessage" {
```
Comment (brettminnie) [inline terraform_modules/sqs-ingress/main.tf:80]: please change this to send_message to match convention

**Review by brettminnie — CHANGES_REQUESTED** [review]

**Comment by TheGrantsta** [comment]
Changes implemented

File: terraform_modules/sqs-ingress/main.tf [hunk 4]
```diff
@@ -93,3 +109,136 @@ resource "aws_sqs_queue_policy" "sendMessage_from_another_account" {
   queue_url = aws_sqs_queue.input_queue[0].id
   policy    = data.aws_iam_policy_document.sendMessage.json
```
Comment (brettminnie) [inline terraform_modules/sqs-ingress/main.tf:110]: this needs to be changed to the new name too

Comment (TheGrantsta) [reply terraform_modules/sqs-ingress/main.tf]: Whoops! Fixed

**Review by brettminnie — APPROVED** [review]

**Pull request merged**